assets/src/components/map.tsx

```diff
@@ -39,6 +40,12 @@ import appData from "../appData"
 import { createControlComponent } from "@react-leaflet/core"
 import "leaflet.fullscreen"
 
+import garages, { Garage } from "../data/garages"
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
same ignores as https://github.com/mbta/skate/blob/f1633ba3f09b1aefaa8544c9ac0a5519003963d7/assets/src/helpers/icon.tsx#L1

Reply:
question(non-blocking): do you know why we import SVG with javascript imports? 

maybe this is this something that gets inlined by webpack?
but if so, why do we want it inlined?

Reply:
I'm not sure why - maybe @lemald knows more?

Reply:
Yeah, that was already established practice when I joined and I'm not sure entirely why it's done the way it is. It probably does have something to do with making sure that it gets added to the Webpack build but it's not clear to me if that's actually the best way to handle things. I'm open to suggestions on changes we could make.